### ISSUE TITLE
Menu Items in selected state are not having 4.5:1 color contrast ratio.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#276]
### Steps to Reproduce

1. Launch Xamarin Workbook application. It will open "Create a new workbook" window.
2. Navigate to Menu items and take the screenshot (Refer Screenshots)
3. Observe the colour contrast ratio of Menu(File, view etc.) and their corresponding menu  items which are not selected state.

### Expected Behaviour
Colour contrast ratio should be minimum 4.5:1

### Actual Behaviour
Colour contrast ratio is lesser than 4.5:1 (i.e., 4:1)
as shown in the attached screen-shots.

### User Impact:
Low vision user will feel difficulty in seeing the text.

### Environment

MAC: macOS High Sierra [10.13.4]
Workbook - 1.4 (master/b3f92f9)

## Log File

[Menu Items colour contrast issues.zip](https://github.com/Microsoft/workbooks/files/1991039/Menu.Items.color.contrast.issues.zip)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS1.4.3
Xamarin-Workbook-MAC
Wipro

**Comment by abock** [comment]
This is a stock macOS control. We do not influence or control the rendering of this control. The system-wide "Graphite" appearance may exhibit 4.5:1 contrast. Regardless, this is not something we can or should address in our code as it comes from macOS itself.

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1829
